docs/glues/herbs2rest.md

```diff
@@ -109,8 +135,50 @@ const testUseCase = (injection) =>
   })
 ```
 
+#### Custom Endpoints
+
+You can configure custom REST endpoints and HTTP verbs for your use cases by modifying the `metadata` object of the use case. To do this, include a `REST` key in the metadata, containing a JavaScript object with the HTTP verb as the key and the desired path as the value.
+
+Supported HTTP verbs are: `GET`, `POST`, `PUT`, and `DELETE`.
+
+The path should be a string starting with a forward slash (`/`), followed by the desired path.
+
+Example:
+
+```javascript
+module.exports = 
+  herbarium.usecases
+    .add(createUser, 'CreateUser')
+    .metadata({
+      group: 'User',
+      operation: herbarium.crud.create,
+      entity: User,
+      REST: { post: '/createuser' } // Changes from POST /User to POST /createuser
+    }) 
+    .usecase
+```
+
+You can also customize the resourceName by providing a string within the REST object. This will update the default route for the use case, adhering to conventions. By default, the resourceName is derived from the group in the metadata.
+
+Example:
+
```

Comment:
Could you please show more examples about the generated route?

Reply:
Sure, done.